worker/socialPoster.js

```diff
@@ -92,21 +92,29 @@ async function getHottestItem ({ models }) {
   return item[0]
 }
 
-async function itemToMessage ({ item }) {
+async function itemToMessage ({ item, postAuthorNostrProfile }) {
   return `${item.title}
 
-by ${item.userName} in ~${item.subName}
+by ${item.userName}${postAuthorNostrProfile ? `, nostr profile: nostr:${postAuthorNostrProfile}` : ''} in ~${item.subName}
```

Comment:
the mention should replace `item.userName`, not be appended:

![2025-04-15-214812_634x259_scrot](https://github.com/user-attachments/assets/e2b8bd32-8814-4df4-b358-455a15d531f8)

Reply:
Was not clarified @ekzyis, I assumed we'd want to know the username on stacker too